docs/index.rst

```diff
@@ -5,11 +5,19 @@ Developer Documentation
 This the landing page for documentation relevant to developers of the Open edX platform.  For other documentation
 resources targeted for edX learners, educators, researchers, and Open edX operators, see `docs.edx.org <docs.edx.org>`_.
 
+.. toctree::
+    :hidden:
+
+    Open edX Proposals <https://open-edx-proposals.readthedocs.io/en/latest/>
+    Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>
```

Comment:
These 2 are already listed below in `Architecture and Best Practices`.

Reply:
Removing this for now and putting it in a separate PR that deals with the sidebar.